tc/core/polyhedral/schedule_tree_elem.cc

```diff
@@ -175,6 +175,17 @@ void ScheduleTreeElemBand::drop(size_t pos, size_t n) {
   TC_CHECK_EQ(nBegin - n, nMember());
 }
 
+isl::multi_union_pw_aff ScheduleTreeElemBand::memberRange(
+    size_t first,
+    size_t n) const {
+  auto list = mupa_.get_union_pw_aff_list();
+  auto space = addRange(mupa_.get_space().domain(), n);
+  auto end = first + n;
```

Comment:
Can we have an out-of-bounds check here?  I understand isl will complain, but `nMember() - end` is going to be some large value, inconvenient to debug

Reply:
> Can we have an out-of-bounds check here?  I understand isl will complain, but
`nMember() - end` is going to be some large value, inconvenient to debug

Added.

skimo